quizapp/src/main/java/billennium/quizapp/resource/question/QuestionDto.java

```diff
@@ -13,7 +13,6 @@
 @NoArgsConstructor
 @SuperBuilder(toBuilder = true)
 public class QuestionDto extends QuestionBaseDto {
-    private Integer timeToAnswer;
```

Comment:
dziwne to DTO, nazywa się Question a znajduje sie w nim lista odpowiedzi

Reply:
Bo reszta info jest w base  zeby nie robic kilku dtos ktore maja te same pola i zrobilem tak zeby dziedziczyły z base a w sobie miali te pola ktore maja rozne